test/inductor/test_aot_inductor_package.py

```diff
@@ -23,17 +25,21 @@
     raise
 
 
-def compile(model, example_inputs, dynamic_shapes, options, device):
+def compile(
+    model, example_inputs, dynamic_shapes, options, device
+) -> AOTICompiledModel:
     ep = torch.export.export(
         model,
         example_inputs,
         dynamic_shapes=dynamic_shapes,
         strict=False,
     )
     gm = ep.module()
-    package_path = torch._inductor.aot_compile(gm, example_inputs, options=options)  # type: ignore[arg-type]
-    compiled_model = load_package(package_path, device)
-    return compiled_model
+    aoti_files = torch._inductor.aot_compile(gm, example_inputs, options=options)  # type: ignore[arg-type]
```

Comment:
We probably should provide another `torch._inductor` API that does compilation and packaging, and eventually deprecate `torch._inductor.aot_compile`.

Reply:
My reasoning is for users who currently rely on `torch._inductor.aot_compile` they really need to migrate to an API returning a package, because asking users to copy around those .so and .cubin files don't make sense. What you said is also valid, so we likely will need both.